addons/mail/static/src/core/common/navigable_list.xml

```diff
@@ -4,29 +4,31 @@
     <t t-name="mail.NavigableList">
         <div class="o-mail-NavigableList bg-view m-0 p-0" t-ref="root" t-att-class="props.class">
             <div t-if="show" class="o-open border" t-on-mousedown.prevent="">
-                <div t-if="props.isLoading" class="o-mail-NavigableList-item">
+                <a href="#" t-if="props.isLoading and props.options.length" class="position-absolute end-0 mt-2 me-3 d-flex align-items-center gap-1">
```

Comment:
Add `role="button"` and remove the `href`. 
`#` could set the scroll position at the top; we certainly don't want that.